transcendence-app/src/chat/chat.controller.ts

```diff
@@ -49,6 +55,25 @@ export class ChatController {
     return chat;
   }
 
+  @Post(':chatId/members')
+  @ApiCreatedResponse({ description: 'Add a member to a chatroom', type: Chat })
+  @ApiBadRequestResponse({ description: 'Bad Request' })
+  @ApiUnprocessableEntityResponse({ description: 'Unprocessable entity' })
+  async createChatRoomMember(
+    @GetUser() user: User,
+    @Param('chatId', ParseUUIDPipe) chatId: string,
+  ) {
```

Comment:
praise: I like how you implemented this endpoint and structured the code.

Reply:
Thanks!